index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'mongo-unit' {
+    /**
+     * @returns Url to connect to the created db.
+     */
+    export function start(opts?: Partial<MongodOptions>): Promise<string>;
+
+    export function stop(): Promise<void>;
+
+    export function getUrl(): string;
+
+    export function load(data: object): Promise<void>;
+
+    export function clear(data: object): Promise<void>;
+
+    export function drop(): Promise<void>;
+
+    export function initDb(url: string, data: object): Promise<void>;
+
+    export function dropDb(url: string): Promise<void>;
+
+    export interface MongodOptions {
+        port: number;
+        dbName: string;
+        dbpath: string;
```

Comment:
It could be interesting to port the codebase to TypeScript, because that would guarantee that the types are in sync with the code. However, it's unfortunately not trivial to set up, which is why I only submitted a typings file.

Reply:
Ok, I see now. I think you've made the right choice here 👍